SourceryRuntime/Sources/AST/AssociatedValue_Linux.swift

```diff
@@ -0,0 +1,148 @@
+//
+// Created by Krzysztof Zablocki on 13/09/2016.
+// Copyright (c) 2016 Pixle. All rights reserved.
+//
+#if !canImport(ObjectiveC)
+import Foundation
+// For DynamicMemberLookup we need to import Stencil,
+// however, this is different from SourceryRuntime.content.generated.swift, because
+// it cannot reference Stencil
+import Stencil
+
+/// Defines enum case associated value
+public final class AssociatedValue: NSObject, SourceryModel, AutoDescription, Typed, Annotated, Diffable, DynamicMemberLookup {
+    public subscript(dynamicMember member: String) -> Any? {
+        switch member {
```

Comment:
This list is not exhaustive, but just to make all unit tests work. It must be autogenerated like `init(coder:)` methods, and I'll address it in #1198